### PR TITLE
Add bulk finder

### PR DESCRIPTION
**Pull request by javierg**
Add bulk find methid

### CODE REVIEW

File: lib/dolly/query.rb [hunk 1]
```diff
@@ -17,8 +17,18 @@ def find *keys
         raise(Dolly::ResourceNotFound)
     end
 
+    def bulk_find *keys_to_find
+      data = {
+        query: { include_docs: true },
```
Comment (ErickFabian) [inline lib/dolly/query.rb]: cant this be an option to be passed?

File: lib/dolly/query.rb [hunk 2]
```diff
@@ -17,8 +17,18 @@ def find *keys
         raise(Dolly::ResourceNotFound)
     end
 
+    def bulk_find *keys_to_find
```
Comment (ErickFabian) [inline lib/dolly/query.rb]: be consistent with parenthesis

File: lib/dolly/query.rb (hunk 1 above)
Comment (javierg) [reply lib/dolly/query.rb]: ok

File: lib/dolly/query.rb [hunk 3]
```diff
@@ -17,8 +17,18 @@ def find *keys
...
+      data = {
+        query: { include_docs: true },
+        keys: keys_to_find.map { |key| namespace_key(key) }
+      }
+
+      res = connection.post("_all_docs", data)
```
Comment (ErickFabian) [inline lib/dolly/query.rb]: single quotes

**Comment by ErickFabian** [comment]
whats the difference between `find_all` and `bulk_find` ?

**Comment by javierg** [comment]
> whats the difference between find_all and bulk_find ?

find_all does a `GET`, while bulk_find does a `POST`, but mainly is that we have `bulk_find` in many projects and will make our lives easier having it

**Review by ErickFabian — APPROVED** [review]
:shipit:

**Pull request merged**